src/features/instrument/Instrument.tsx

```diff
@@ -101,24 +164,51 @@ export const Instrument = () => {
     }
   }
 
-  // @TODO: check arent a beter way to write each thing now (always create new player?), also check updates in tone
-  function playSample(sampleName: string) {
-    new Player({
-      url: sampleName,
-      autostart: true,
-    }).chain(mediumVolume, Destination);
-  }
+  return (
+    <>
+      <Button onClick={toggleNoise}>💨 Noise</Button>
+    </>
+  );
+}
+
+// rethink is interface is needed (maybe due to consistence + pretty long one)
+interface MarioSamplesProps {
```

Comment:
**Do you create interfaces when it's only 1 prop?**
I didn't want, but added because of:
- consistency In this file (for other components I added, those had 2 props)
- type of `playSample` is pretty long

**What's your thinking flow?**

Reply:
You can do them inline, but sometimes it's cleaner to just do a 1-liner above the function, especially when you're typing a function like you are here.

```tsx
// inline is a lil messy
export const MarioSamples = ({ playSample }: {playSample: (string) => void}) => {

// The linter may be ok with this, it may just reformat it to what you currently have.
interface MarioSamplesProps {playSample: (string) => void};

export const MarioSamples = ({ playSample }: MarioSamplesProps) => {
```